File: 1. Array/Largest/Largest.java
package Largest;
import java.util.Scanner;

public class Largest {
    public static final Scanner scan = new Scanner(System.in);
    public static void main(String[] args) {
        int arr[] = new int[]{10, 20, 30, 40, 50};
        int ans = FindLargestElement(arr);
        System.out.println(ans);
    }
    public static int FindLargestElement(int arr[]){
        int max = arr[0];
        for (int i=0; i<arr.length; i++){
            if(arr[i] > max) {
                max = arr[i];
            }
        }
        return max;
    }
}

File: 1. Array/Largest/Largest.go
package main

import "fmt"

func main() {
	arr := []int{10, 20, 30, 40, 50}
	var ans = FindLargestElement(arr)
	fmt.Println(ans)
}

var FindLargestElement = func(arr []int) int {
	var max = arr[0]
	for _, value := range arr {
		if value > max {
			max = value
		}
	}
	return max
}
